src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8002,4 +8002,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UninitializedNonNullableBackingField_Title" xml:space="preserve">
     <value>Non-nullable property must contain a non-null value when exiting constructor. Consider adding the 'required' modifier, or declaring the property as nullable, or adding '[field: MaybeNull, AllowNull]' attributes.</value>
   </data>
+  <data name="WRN_AccessorDoesNotUseBackingField" xml:space="preserve">
+    <value>The '{0}' accessor of property '{1}' should use the backing 'field' because the other accessor is using it.</value>
```

Comment:
Honestly, `the backing 'field'` reads very awkwardly. Maybe just `should use 'field'`?